paddlenlp/transformers/auto/configuration.py

```diff
@@ -230,7 +229,8 @@ def from_pretrained(cls, pretrained_model_name_or_path: str, *model_args, from_h
                 logger.warning(f"loading legacy config file<{cls.legacy_config_file}> ...")
                 community_config_path = legacy_community_config_path
 
-            default_root = os.path.join(MODEL_HOME, pretrained_model_name_or_path)
+            # we already handled the cache_dir logic through resolve_cache_dir
+            default_root = cache_dir
             resolved_config_file = get_path_from_url(community_config_path, default_root)
```

Comment:
```suggestion
            # we already handled the cache_dir logic through resolve_cache_dir
            resolved_config_file = get_path_from_url(community_config_path, cache_dir)
```

在这里`default_root`只在`get_path_from_url`函数中使用，所以建议没必要用`default_root` 来中转。

而且在其他分支中也都是使用的 `cache_dir` 来处理，可是这里单独使用default_root 来中转处理，感觉没必要。

Reply:
我看了下其他`modeling` 和 `tokenizer` 模块都是存在类似问题。